widgetssdk/src/main/java/com/glia/widgets/core/visitor/VisitorInfoUpdate.java

```diff
@@ -32,7 +36,7 @@ public void setNote(String note) {
         this.note = note;
     }
 
-    public void setCustomAttributes(LinkedTreeMap<String, String> customAttributes) {
+    public void setCustomAttributes(Map<String, String> customAttributes) {
```

Comment:
Is it breaking change? could we avoid this?

Reply:
Actually not, because `LinkedTreeMap` is implementing the `Map` interface, I modified setters to accept the `Map` interface type and provided a second method for the getter to return `Map`.These changes will not affect the existing code.